packages/subject-metadata-controller/src/SubjectMetadataController.ts

```diff
@@ -154,7 +154,7 @@ export class SubjectMetadataController extends BaseControllerV2<
   /**
    * Stores domain metadata for the given origin (subject). Deletes metadata for
    * subjects without permissions in a FIFO manner once more than
-   * {@link SubjectMetadataController.subjectCacheLimit} distinct origins have
+   * {@link SubjectMetadataController.#subjectCacheLimit} distinct origins have
```

Comment:
Is this worth linking to if it's a private method? (Such a link may not be accessible in the generated docs.)

Reply:
There may be other examples of this that I came across but didn't comment on. I'll take a look later to see if I can find them.

Reply:
Good point, probably not. I think either way (even with the `private` modifier), this wouldn't be included in the doc.